integration-test/tests/incontext-signup.extension.spec.js

```diff
@@ -62,7 +62,7 @@ test.describe('chrome extension', () => {
         await emailPage.assertExtensionPixelsCaptured(['incontext_show', 'incontext_dismiss_persisted'])
     })
 
-    test('should allow tooltip to be closed', async ({page}) => {
+    test('should allow tooltip to be closed @flaky', async ({page}) => {
```

Comment:
marking as flaky for now, since it was observed as such locally (before this change). after a number of CI runs we can decide if these changes have helped, or not :)